src/Skeletor/Frameworks/Framework.php

```diff
@@ -86,11 +87,11 @@ public function setVersion(string $version)
      */
     public function getPath(string $path)
     {
-        if(array_key_exists($path, $this->paths)){
-            return $this->paths[$path];
+        if(array_key_exists($path, $this->paths) === false){
```

Comment:
@titopixelfusion I disagree, I find the `!` is easily missed when skimming through code. I prefer the `=== false` for readability.